packages/office-ui-fabric-react-experiments/tslint.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": [
+    "office-ui-fabric-react-tslint"
+  ],
+  "rules": {
+    "max-line-length": [
```

Comment:
Can you please remove all of the rules here? rules should be empty to be more strict. The only reason we have overrides elsewhere is so that we can gradually transition to the rules defined in the oufr-tslint project.